delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/projects/ProjectRejection.scala

```diff
@@ -101,10 +101,15 @@ object ProjectRejection {
 
   implicit private[model] val projectRejectionEncoder: Encoder.AsObject[ProjectRejection] =
     Encoder.AsObject.instance { r =>
-      val tpe = ClassUtils.simpleName(r)
+      val tpe     = ClassUtils.simpleName(r)
+      val default = JsonObject.empty.add(keywords.tpe, tpe.asJson).add("reason", r.reason.asJson)
       r match {
         case WrappedOrganizationRejection(rejection) => rejection.asJsonObject
-        case _                                       => JsonObject.empty.add(keywords.tpe, tpe.asJson).add("reason", r.reason.asJson)
+        case IncorrectRev(provided, expected)        =>
```

Comment:
I guess `IncorrectRev` and `RevisionNotFound` should be addressed in all rejections, not only in `ProjectRejection `